### PR TITLE
FEAT: Use only RPC POST

### PR DESCRIPTION
**Pull request by x8x**

### CODE REVIEW

File: libs/ethereum.red [hunk 1]
```diff
@@ -28,7 +27,7 @@ eth: context [
 		mul256 n GWei-ratio
 	]
 
-	pad64: function [data [string! binary!]][
+	pad64: func [data [string! binary!] /local c len n][
```
Comment (dockimbel) [inline libs/ethereum.red]: Why this change?

File: libs/ethereum.red [hunk 2]
```diff
@@ -37,7 +36,7 @@ eth: context [
 		data
 	]
 
-	parse-balance: function [amount][
+	parse-balance: func [amount [string!] /local n][
```
Comment (dockimbel) [inline libs/ethereum.red]: Why this change?

File: libs/ethereum.red (hunk 1 above)
Comment (x8x) [reply libs/ethereum.red]: Consistency and speed and new comers should better use `func` to avoid headaches later. I remember you saying to have been bitten by this. Your call, it looks nicer with `function`

File: libs/ethereum.red (hunk 2 above)
Comment (x8x) [reply libs/ethereum.red]: Same as above

File: libs/ethereum.red (hunk 1 above)
Comment (dockimbel) [reply libs/ethereum.red]: Right, but we are not "newcomers", so `function` is what we want to use there. ;-) For performance, unless your code is creating millions of functions, there won't be any impact on apps performance.

Comment (x8x) [reply libs/ethereum.red]: I didn't mean to alter the code, I just thought it was a leftover from quick development because you guys are so busy. But I appreciate your comment, I will revert it and never do it again! :-)

File: libs/rpc.red [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+context [
+
+	headers: compose [
+		Content-Type: "application/json"
+		Accept: "application/json"
+		User-Agent: (
+			form reduce [
+				{Red Wallet version}
+				read %version.red
```
Comment (qtxie) [inline libs/rpc.red]: The executable must be along with the %version.red file. Can we use macro here? so we get the version in compiling time.

Comment (x8x) [reply libs/rpc.red]: :+1: Will do in the evening, thx!

**Comment by x8x** [comment]
Wondering if `libs/rpc.red` should be integrated in the `eth` object in `libs/ethereum.red` instead of separate file..
